app/Domain/Mail/MailManager.php

```diff
@@ -101,4 +101,22 @@ private function getIntegrationVariables(Contact $contact, Integration $integrat
             'type' => $integration->type->value,
         ];
     }
+
+    /**
+     * To optimize email credits and prevent spamming we check that the same email is not sent multiple times to the same e-mail address
+     * @return Contact[]
+     */
+    private function getContacts(Integration $integration): array
+    {
+        $uniqueContacts = [];
+
+        foreach($integration->contacts() as $contact) {
+            // Because sometimes the technical contacts get some additional info this contact gets preference when matching email addresses are found
```

Comment:
We sent a satisfaction survey in one of the mails only to the Technical contact. So extra info in the email.

Reply:
There are examples where the technical, functional, and contributor have the same email address. With this loop this would result in a list of 2 email addresses that are the same.

Reply:
I guess "additional info" should be "survey email"